ui/src/app-bar/app-bar.scss

```diff
@@ -58,14 +58,36 @@ $appbar-height: 48px;
             color: $color-blue-light;
         }
 
+        &--infra-model {
+            @include typography-body-strong;
+
+            display: inline-block;
+            line-height: $appbar-height;
+            height: $appbar-height - 4px;
+            padding: 0 20px;
```

Comment:
Nämä `&--infra-model`:in määritykset näyttäisivät olevan tätä lukuunottamatta identtiset tuon `&__link`:in kanssa, joten tätä luokkaa käyttäville komponenteille voisi vaan antaa molemmat luokat, ja täällä yliajaa pelkästään tämä padding.

Reply:
Hyvä vinkki, kiitos! Korjattu